jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/thread/ThreadPool.java

```diff
@@ -87,4 +88,42 @@ default ThreadPoolBudget getThreadPoolBudget()
             return null;
         }
     }
+
+    /**
+     * <p>Execute a task immediately without queueing.   This may use a
+     * {@code ReservedThread}, a {@code Virtual Thread}, a call to {@link Invocable#invokeNonBlocking(Runnable)},
+     * a newly spawned thread, or direct execution.
+     *
+     * @param executor An executor that may be used
+     * @param task The task that must be executed.
+     */
+    static void mustExecute(Executor executor, Runnable task)
```

Comment:
The name of this method lacks the fact that it's about executing now, without any form of queuing.

Maybe rename it to `executeNow` or `executeWithoutQueuing`?

Reply:
How about `executeImmediately`

Reply:
I'm happy with `executeImmediately`.